.github/workflows/test-coverage.yml

```diff
@@ -236,6 +236,27 @@ jobs:
           coverage-artifact-name: "coverage_encryption"
           coverage-path: coverage.txt
 
+  # This job runs the database with telemetry tests using default configuration, on linux.
+  test-telemetry:
+    name: Test telemetry job
+
+    runs-on: ubuntu-latest
+
+    env:
+      GO_FLAGS: -tags=telemetry
+
+    steps:
+      - name: Checkout code into the directory
+        uses: actions/checkout@v4
+
+      - name: Setup defradb
+        uses: ./.github/composites/setup-defradb
+
+      - name: Test coverage & save coverage report in an artifact
+        uses: ./.github/composites/test-coverage-with-artifact
+        with:
+          coverage-artifact-name: "coverage_encryption"
```

Comment:
issue: This typo is very problematic as it will fail all reports due to duplicate artifact name (or overwrite which was the old behavior). Please fix the name to something relevant.

Reply:
nice catch! should be fixed now